src/dc_mimefactory.rs

```diff
@@ -1315,3 +1220,20 @@ unsafe fn is_file_size_okay(context: &Context, msg: &Message) -> bool {
 
     file_size_okay
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_slice_to_clist() {
+        unsafe {
+            let input = vec!["foo".into(), "bar".into()];
+            let clist = slice_to_clist(&input[..]);
+
+            assert_eq!((*clist).count, 2);
+
+            clist_free(clist);
```

Comment:
Please make sure to have a way of freeing the strings, this is leaking memory, as `clist_free` does __not__ free its content!

ref: https://github.com/dignifiedquire/mmime/blob/master/src/clist.rs#L78

Reply:
It is test. Why it matters?

Reply:
Because it will inform others looking how to use this. Also I am not sure that the other use cases of using the slice to clist methods take care of this.

Reply:
Reasonable. Added `free` loop.

In the only call site, this function replaces `dc_str_to_clist`, which also returns allocated clist with allocated content.